### ISSUE TITLE
Bad naming for message classes of the same name but from different packages yielding collisions

### ISSUE THREAD

**Issue by alexmtrmd** [akka/akka-grpc#1625]
https://github.com/akka/akka-grpc/blob/756a185d129ff8dc988b82679b759440944dbe53/codegen/src/main/twirl/templates/JavaCommon/ApiInterface.scala.txt#L30

is already defined as value 
```
 val SendOrderRequestSerializer = new ScalapbProtobufSerializer(...
```

**Comment by raboof** [comment]
Do we ever generate code that imports from both those packages/classes at the same time? Could you show an example of what you mean?

**Comment by raboof** [comment]
Or do you mean multiple messages with the same names (in different packages) within the same service? I thin that might indeed cause trouble.

**Comment by jroper** [comment]
I just encountered this myself. It's when you have multiple messages with the same names imported from different packages.

**Comment by jroper** [comment]
Fixed in #1643.
